CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 - Pass `pip_args` to `shared_libs.upgrade()`
 - Fallback to user's log path if the default log path (`$PIPX_HOME/logs`) is not writable to aid with pipx being used for multi-user (e.g. system-wide) installs of applications
 - Fix wrong interpreter usage when injecting local pip-installable dependencies into venvs
-- add pre-commit hook support
+- Add pre-commit hook support
+- Fix FileNotFoundError when updating package cache on Windows
```

Comment:
Can you move these two entrys to the top?

Reply:
Hi @chrysle, sure thing - thanks for the guidance, this is my first contribution here 😄. Update made 👍